languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -8426,6 +8432,15 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
             <example>as bagagens não convencionais agora desaparecerão</example>
         </antipattern>
 
+        <antipattern>
```

Comment:
The key blocker of agreement here is the verb in the first-person singular, not the whole construction. I'd rethink this.